File: CourseWorksPortal/CourseWorksPortal/Models/CourseWork.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.AspNetCore.Http;

namespace CourseWorksPortal.Models
{
    public class CourseWork
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string File { get; set; }
        public string File_name { get; set; }
    }
}

File: CourseWorksApp/app/src/main/java/com/safronov/courseworksapp/Models/CourseWork.java
package com.safronov.courseworksapp.Models;

public class CourseWork {

    public int id;
    public String name;
    public String description;
    public String file;
    public String file_name;

    public CourseWork(int id, String name, String description, String file_name) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.file_name = file_name;
    }
}
